src/input/MeshAttributes.cpp

```diff
@@ -206,6 +213,10 @@ void MeshAttributes::set_velocity_aware_meshing() {
                           << "with half sizes"
                           << "x =" << cuboid.halfSize[0] << "y =" << cuboid.halfSize[1]
                           << "z =" << cuboid.halfSize[2];
+      if (abs(cuboid.rotationZ) > 0.0) {
```

Comment:
Should be std::abs

Reply:
See here for reasoning: https://stackoverflow.com/a/21393170
Using abs can lead to compatibility problems